src/services/challenges_repository.js

```diff
@@ -46,14 +47,17 @@ export default class ChallengesRepository {
     this.activeChallengesCache.applyIncomingChallengeEvents(startedChallenges, resolvedChallenges, timedOutChallenges);
   }
 
-  async updateBlockInfo() {
+  async getBlockInfo() {
     const challengeDuration = await this.configWrapper.challengeDuration();
```

Comment:
We can save one additional call to parity if we did this only in case when there is no lastSavedBlock (rhs of below conditional)